query.go

```diff
@@ -235,18 +236,27 @@ func (q *Query) setBasicAuth(req *http.Request) string {
 }
 
 func (q *Query) makeRequest(req *http.Request) (*http.Response, error) {
-	presto_user := q.setBasicAuth(req)
+	prestoUser := q.setBasicAuth(req)
 	req.Header.Add("User-Agent", userAgent)
-	req.Header.Add("X-Presto-User", presto_user)
+	req.Header.Add("X-Presto-User", prestoUser)
 	req.Header.Add("X-Presto-Catalog", q.catalog)
 	req.Header.Add("X-Presto-Schema", q.schema)
 	req.Header.Add("X-Presto-Source", q.source)
 
+	tr := &http.Transport{ //Fixes cert x509 signed by unknown authority
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	client := &http.Client{
+		Timeout:   15 * time.Second,
```

Comment:
better set set connect timeout here, whole request can easily take more than this (long query)